_data/releases.csv

```diff
@@ -290,3 +290,4 @@ v21.2.14,v21.2,2022-08-01,false,False,Production,False,go1.16,8f4dc943d39780a07d
 v22.1.6,v22.1,2022-08-23,false,False,Production,False,go1.17,760a8253ae6478d69da0330133e3efec8e950e4e,cockroachdb/cockroach,true
 v22.2.0-alpha.1,v22.2,2022-08-30,false,False,Testing,False,go1.17,3688055c09cd32f1186624a1ccc3fe35e5a0cffd,cockroachdb/cockroach-unstable,true
 v21.2.15,v21.2,2022-08-29,false,False,Production,False,go1.16,2326e63d4a9ad62b19056f90f938a00482cbf56a,cockroachdb/cockroach,false
+v22.2.0-alpha.2,v22.2,2022-09-06,false,False,Testing,False,go1.17,d676dcb809083388a3a280a7f9d5c8230b1682e6,cockroachdb/cockroach-unstable,true
```

Comment:
v22.2 is built on `go1.18`:

```
$ cockroach --version
   cockroach version details:
   Build Tag:        v22.2.0-alpha.1
   Build Time:       2022/08/30 23:02:58
   Distribution:     CCL
   Platform:         darwin amd64 (x86_64-apple-darwin19)
   Go Version:       go1.18.4
   C Compiler:       Clang 10.0.0
   Build Commit ID:  3688055c09cd32f1186624a1ccc3fe35e5a0cffd
   Build Type:       release
   (use 'cockroach-22.2 version --build-tag' to display only the build tag)
```

Both `alpha.1` and `alpha.2` entries indicate `go1.17`

Reply:
Looks like someone from Engineering didn't update `go.mod`, which is where I calculate the go version.

Reply:
Ah gotcha. You wanna remind them, or want me to?